astropy/utils/console.py

```diff
@@ -815,10 +833,13 @@ def work(i):
                     if (i % chunksize) == 0:
                         bar.update(i)
             else:
-                p = multiprocessing.Pool(
+                p = multiprocessing.get_context(
+                            multiprocessing_start_method
+                ).Pool(
                     processes=(int(multiprocess) if multiprocess is not True else None))
                 for i, result in enumerate(
-                    p.imap_unordered(function, items, chunksize=chunksize)):
+                    p.imap_unordered(function, items, chunksize=chunksize)
+                ):
```

Comment:
I don't find this isolated `):` much of an improvement... In PEP-8, it seems used primarily for list and tuples.

Reply:
I picked up the habit from black's style - it ensures that the indentation level of the arguments is one more than the thing they're arguments to, and that the material after the : is indented one more than the thing with the :, but also that there is a clear visual distinction of where the one turns into the other.

The other natural option is to indent the arguments one tab more. <shrug /> All I could find in astropy's style guide was "please follow pep8", which doesn't help much.

Reply:
Ah, yes, now I see why you have it sometimes but not always...  I'm still far from sure I would use it myself, but will withdraw the complaint!